### PR TITLE
Improved dashboard

### PR DESCRIPTION
**Pull request by normadn**

### CODE REVIEW

File: components/shared/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        {({ open }) => (
+          <>
+            <div className="mx-auto max-w-7xl px-2 sm:px-4 lg:px-8">
+              <div className="relative flex h-16 items-center justify-between lg:border-b lg:border-indigo-400 lg:border-opacity-25">
+                <div className="flex items-center px-2 lg:px-0">
+                  <div className="flex flex-shrink-0">
+                    <img
+                      className="block h-8"
+                      src="/FRENS_logo_white.png"
+                      alt="Your Company"
```
Comment (heeckhau) [inline components/shared/Header.tsx:31]:
```suggestion
                      alt="Staking with Frens"
```

File: components/shared/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    />
+                    <img
+                      className="block h-8 pl-2"
+                      src="/FRENS_writing_white.png"
+                      alt="Your Company"
```
Comment (heeckhau) [inline components/shared/Header.tsx:36]:
```suggestion
                      alt="Staking with Frens"
```

File: pages/dashboard.tsx [hunk 3]
```diff
@@ -1,17 +1,48 @@
...
-import { ActionButtons } from "#/components/dashboard/ActionButtons";
+import { PlusSmallIcon } from "@heroicons/react/20/solid";
+import Link from "next/link";
+
+const secondaryNavigation = [
+  { name: "Last 7 days", href: "#", current: true },
+  { name: "Last 30 days", href: "#", current: false },
+  { name: "All-time", href: "#", current: false },
+];
+const stats = [
```
Comment (heeckhau) [inline pages/dashboard.tsx:18]: hardcoded?
TODO: replace with actual values

File: pages/dashboard.tsx [hunk 4]
```diff
@@ -1,17 +1,48 @@
...
+  {
+    name: "Pool Shares #",
+    value: "40",
+  },
+  {
+    name: "ETH Deposited",
+    value: "34.123",
+  },
+  {
+    name: "ETH claimalbe",
```
Comment (heeckhau) [inline pages/dashboard.tsx:34]: typo: claimable

**Review by heeckhau — CHANGES_REQUESTED** [review]
Awesome work! Looks really nice and clean! ❤️ 

If we replace the hardcoded statistics with the actual values, this is ready to merge

File: components/shared/Header.tsx (hunk 1 above)
Comment (normadn) [reply components/shared/Header.tsx]: done

File: pages/dashboard.tsx (hunk 4 above)
Comment (normadn) [reply pages/dashboard.tsx]: fixed

File: components/shared/Header.tsx (hunk 2 above)
Comment (normadn) [reply components/shared/Header.tsx]: fixed

File: pages/dashboard.tsx (hunk 3 above)
Comment (normadn) [reply pages/dashboard.tsx]: u fixed that :)

**Pull request merged**